<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="paused" default="false" />

  <!-- Launch Gazebo with empty world-->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="gui" value="true" />
    <arg name="paused" value="$(arg paused)" />
  </include>

    <node name="cam_sp" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen" args="-sdf -model camera -file $(find calibration_gazebo)/sdf/perspective.sdf -x 0 -y 0. -z 3 -R 0.1 -P 0.3" />
    
    <!-- launch camera calibrator  -->
    <node name="calibrator" pkg="camera_calibration" type="cameracalibrator.py" args="-q 0.03 -s 8x6" output="screen" ns="camera"/>

</launch>
